docs/install.md

```diff
@@ -78,6 +78,8 @@ Requirements {#netcdf_requirements}
   * HDF5 1.10.1 or later.
 * zlib 1.2.5 or later (for netCDF-4 compression)
 * curl 7.18.0 or later (for DAP remote access client support)
+* For paralle I/O support on classic netCDF files
+  * PnetCDF 1.6.0 or later
```

Comment:
I test on 1.8.0 and 1.9.0, I don't know what Ward tests with. 1.6.0 seems pretty downrev for pnetcdf.

Reply:
It is just a minimum requirement.
I believe when NetCDF incorporated PnetCDF, PnetCDF was 1.6.0 young.